src/Sarif.Viewer.VisualStudio.sln

```diff
@@ -27,36 +27,18 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Sarif.Viewer.VisualStudio.Interop", "Sarif.Viewer.VisualStudio.Interop\Sarif.Viewer.VisualStudio.Interop.csproj", "{9D407555-D268-461E-BFAF-4251B0FBF7C9}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Sarif.Sarifer", "Sarif.Sarifer\Sarif.Sarifer.csproj", "{F3E3199F-C005-4A14-B96B-974896986EDC}"
-EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Sarif.Sarifer.UnitTests", "Sarif.Sarifer.UnitTests\Sarif.Sarifer.UnitTests.csproj", "{52579A83-C0BF-4307-95CD-39C388070736}"
-EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Sarif", "sarif-pattern-matcher\Src\sarif-sdk\src\Sarif\Sarif.csproj", "{BF73BD60-7DBC-4DA0-A13D-13246EEF8A93}"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Sarif.Driver", "sarif-pattern-matcher\Src\sarif-sdk\src\Sarif.Driver\Sarif.Driver.csproj", "{ACFA4370-A4EC-4252-99E7-A7FA489126DB}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "RE2.Managed", "sarif-pattern-matcher\Src\RE2.Managed\RE2.Managed.csproj", "{A4DF60A6-F128-4BFF-AB1E-E042A3279073}"
-EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Sarif.Converters", "sarif-pattern-matcher\Src\sarif-sdk\src\Sarif.Converters\Sarif.Converters.csproj", "{BA882944-7D32-4543-959A-964B7795F11E}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Sarif.PatternMatcher", "sarif-pattern-matcher\Src\Sarif.PatternMatcher\Sarif.PatternMatcher.csproj", "{141A7597-CC51-4EAC-A168-D2691149AB5B}"
```

Comment:
>"Sarif.PatternMatcher" [](http://example.com/codeflow?start=52&length=22)

if we move the sarifer to another repo, we can also remove the "sarif-pattern-matcher" submodule, and add "sarif-sdk" submodule